share/backend_config.ini

```diff
@@ -19,7 +19,7 @@ interpreter       = perl
 max_zonemaster_execution_time            = 300
 number_of_processes_for_frontend_testing = 20
 number_of_processes_for_batch_testing    = 20
-
+#age_reuse_previous_test=600
 #
```

Comment:
I think this line should be blank to make a clear separation with the following WARNING clause that is related to `maximal_number_of_retries`.

Reply:
Fixed and made the warning clearer. For how long should `maximal_number_of_retries` be considered to be experimental?

Reply:
No idea... I found PR #544 where this parameter has been added. I understand that this option has not been fully tested (as the warning says) and that there might be issues with parallelism.